examples/data_cnv/job.json

```diff
@@ -1,6 +1,6 @@
 {
     "input_files": {
-        "/work/slac/data/hps_010118_10k_events.evio.00000": "data_events.evio"
+        "/sdf/group/hps/data/physrun2019/hps_010104/hps_010104.evio.00114": "data_events.evio"
```

Comment:
@sarahgaiser Here is an input file in an example with a path on sdf.

Reply:
outdated commit